source/blog/2017-02-16-hyperconverged-ovirt-with-ovn.md

```diff
@@ -0,0 +1,355 @@
+```
+cat <<EOF >> /etc/hosts
+192.168.1.202 demovm2.localdomain demovm2
+192.168.1.203 demovm3.localdomain demovm3
+EOF
+```
+At this point we can go to the hosts tab and choose new host setting name=address=*demovm2.localdomain* and its root password in the general tab and *deploy* in the hosted-engine tab as for pictures 16 and 17.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture16.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture16.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture16.png")
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture17.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture17.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture17.png")
+Repeat it for the third host and wait for both of them to come up as for picture 18. The hosts are ready for hosted-engine when they are shown with a little crown: the host where the engine VM is running is shown with a gold crown, the other two with silver crown.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture18.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture18.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture18.png")
+The running status of hosted-engine could be checked also from cockpit running on any of the participants hosts as for picture 19.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture19.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture19.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture19.png")
+The last important step is to check the HA configuration of each host:
+```
+[root@demovm3 ~]# cat /etc/ovirt-hosted-engine/hosted-engine.conf 
+ca_cert=/etc/pki/vdsm/libvirt-spice/ca-cert.pem
+gateway=192.168.1.1
+iqn=
+conf_image_UUID=4e421013-73ed-452d-b2ed-c9f76e4b3bc6
+ca_cert=/etc/pki/vdsm/libvirt-spice/ca-cert.pem
+sdUUID=ffc2409e-680b-4d4a-be0b-9c2b381887cc
+connectionUUID=937c723a-1232-4a24-8612-2044e454a232
+conf_volume_UUID=f6258dac-da0d-499a-a5db-0de0821f86cc
+user=
+host_id=3
+bridge=ovirtmgmt
+metadata_image_UUID=ab5ea2f9-52d2-4419-bec6-f48915ebb8c2
+spUUID=00000000-0000-0000-0000-000000000000
+mnt_options=backup-volfile-servers=192.168.1.202:192.168.1.203
+fqdn=enginevm.localdomain
+portal=
+vm_disk_id=6c4b4fe0-c1f8-4f76-b9f8-757c15d8d046
+metadata_volume_UUID=ca58722f-3f89-4f6a-9c5e-e19aa4166e2f
+vm_disk_vol_id=71d3095b-2eae-489d-9836-7522309b0003
+domainType=glusterfs
+port=
+console=vnc
+ca_subject="C=EN, L=Test, O=Test, CN=Test"
+password=
+vmid=f478697b-52f2-42f6-9f03-e45639952986
+lockspace_image_UUID=1eee643d-7dd5-4bad-8ddc-44cfdc516449
+lockspace_volume_UUID=30acc22f-623b-48bb-a244-62181a60c18c
+vdsm_use_ssl=true
+storage=192.168.1.201:/engine
+conf=/var/run/ovirt-hosted-engine-ha/vm.conf
+```
+having 'mnt_options=backup-volfile-servers=192.168.1.202:192.168.1.203' there means that every host could run the hosted-engine VM mounting the gluster storage domain from any of the hosts in the pool without a single point of failure.
+Tho hosts over the three are required to be up in order to run the engine VM.
+
+## oVirt OVN Provider
```

Comment:
This is exactly where I'll split, with something:
'In part 2, I'll show how to add the OVN provider to this setup.'